src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchConnectorIT.java

```diff
@@ -48,6 +49,23 @@ public static void setupBeforeAll() {
     container.start();
   }
 
+  @Test
+  public void testBatchByByteSize() throws Exception {
+    int approximateRecordByteSize = 60;
```

Comment:
looks good, but maybe add how this was calculated in case the record size changes